### DISCUSSION TITLE
寻求 QueryPageOptions 作为 webApi 参数的可序列化方法

### DISCUSSION THREAD

**Discussion by kimdiego2098** [dotnetcore/BootstrapBlazor#4693] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

![image](https://github.com/user-attachments/assets/850aef78-56ce-49c6-9758-14530cd1c1bb)

``IFilterAction`` 直接序列化并反序列化后，因为没有相关公开属性保存，所以在API中无法获取到实际的过滤条件，有没有什么办法可以实现Table查询条件传到API接口中的方法

### Describe the solution you'd like

实现Table查询条件传到API接口中的方法

**Comment by bb-auto** [comment]
@kimdiego2098 Thank you for contacting us. We will give feedback later.

**Comment by kimdiego2098** [comment, marked as answer]
已解决，可通过  ``FilterKeyValueAction = QueryPageOptions.ToFilter()``获取可序列化的 ``FilterKeyValueAction``
